app/src/main/res/values/dimens.xml

```diff
@@ -8,8 +8,9 @@
   <dimen name="abc_dropdownitem_text_padding_left">8dip</dimen>
   <dimen name="abc_dropdownitem_text_padding_right">8dip</dimen>
   <dimen name="kiwix_search_widget_layout_id_height">50dp</dimen>
-  <dimen name="file_management_no_files_padding_bottom">75dp</dimen>
-  <dimen name="file_management_no_files_text_size">20sp</dimen>
+  <dimen name="custom_text_view_text_size">20sp</dimen>
```

Comment:
these have the wrong names now and honestly we don't particularly need to define these seeing as they are only used in 1 place - the style - right now